v19.2/cockroachcloud-create-your-cluster.md

```diff
@@ -0,0 +1,95 @@
+
+Once you have signed into your CockroachCloud account, you can create CockroachDB clusters using the CockroachCloud Console.
+
+{{site.data.alerts.callout_info}}
+Before creating a cluster, check that you have the correct organization selected. Click the drop down in the top right corner to select a specific organization. For more information on creating and managing organizations, see Organizations.
+{{site.data.alerts.end}}
+
+## Step 1. Access the Create new cluster page
+
+1. Sign in to your CockroachCloud account.
+2. On the **Overview** page, click **Create Cluster**.
+
+##Step 2. Select the cloud provider
+
+On the **Create new cluster** page, select either **Google Cloud** or **AWS** as your preferred cloud provider.
+
+The choice of the cloud provider decides the performance characteristics and price per node. For optimal performance, choose the cloud platform on which your application is deployed. For pricing comparison, refer to the following table:
+
+Hardware configuration	| GCP Pricing (per node, per month)	| AWS Pricing (per node, per month)
+----------|------------|------------
+Small (2 vCPU, 60 GB disk) |	$75	| $180
+Medium	(4 vCPU, 100 GB disk) | $300 | $400
+Large (4 vCPU, 250 GB disk) | $540	| $700
+
+## Step 3. Select the region and number of nodes
+
+### Select region
+
+For optimal performance, select the cloud provider region in which you are running your application. For example, if your application is deployed in GCP's `us-east1` region, select `us-east1` for your CockroachCloud cluster.
+
+### Select number of nodes
+
+- For single-region application development and testing, you may create a one-node cluster.
+- For single-region production deployments, we recommend a minimum of three nodes. The number of nodes also depends on your storage capacity requirements. See Example for further guidance.
+- For multi-region deployments, [contact us](mailto:sales@cockroachlabs.com).
+
+As of now, you can add a maximum of 24 nodes to your cluster.
+
+## Step 4. Select the hardware configuration
+
+While selecting the hardware configuration, consider the following factors:
+
+Factor | Description
+----------|------------
+Capacity | Total raw data size you expect to store without replication.
+Replication | The default replication factor for a CockroachCloud cluster is 3.
+Buffer | Additional buffer (overhead data, accounting for data growth, etc.).
+Compression | The percentage of savings you can expect to achieve with compression. With Snappy, the default algorithm, we typically see about a 40% savings on raw data size.  
+
+To change the hardware configuration after the cluster is created, you will have to contact us.
+
+The choice of hardware per node determines the throughput and performance characteristics of your cluster.
+
+The following table gives the performance characteristics for YCSB and TPC-C workloads per hardware configuration:
+
+<Performance characteristics table>
+<Also include IOPS numbers for each config>
```

Comment:
I'd either not include iops number or if we do include IOPS number specify what that means in GCP v AWS. In GCP we dont get to pick IOPS. In AWS, these are provisioned IOPS.

Reply:
Punting this to the hardware guidance/perf tests PR.